.github/workflows/pkg.pr.new.yml

```
@@ -0,0 +1,21 @@
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - run: corepack enable
+      - uses: oven-sh/setup-bun@v2
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)